Makefile

```diff
@@ -14,18 +14,29 @@ clean: ; $(info cleaning previous builds...)	@
 # Setup Targets
 #------------------------------------------------------
 
-prereqs: 											## Verify that required utilities are installed
-	@echo -- $@ for MBG Project--
-	@go version || (echo "Please install GOLANG: https://go.dev/doc/install" && exit 1)
-
-test-prereqs: prereqs
-	@which goimports || (echo "Please install goimports: https://pkg.go.dev/golang.org/x/tools/cmd/goimports" && exit 1)
-	$(GO) install github.com/mfridman/tparse@latest
-	@kubectl version --client || (echo "Please install kubectl: https://kubernetes.io/docs/tasks/tools/" && exit 1)
-	@docker version --format 'Docker v{{.Server.Version}}' || (echo "Please install Docker Engine: https://docs.docker.com/engine/install" && exit 1)
-	@kind --version || (echo "Please install kind: https://kind.sigs.k8s.io/docs/user/quick-start/#installation" && exit 1)
-	@python3 --version || (echo "Please install python3 https://www.python.org/downloads/ "&& exit 1)
+#-- development tooling --
+.PHONY: prereqs prereqs-force
 
+prereqs: ; $(info installing dev tooling...) 
+	./hack/install-devtools.sh
+
+prereqs-force: ; $(info force installing dev tooling...)
+	./hack/install-devtools.sh --force
+
+.dev-container: Containerfile.dev
+	docker build -f Containerfile.dev -t $(IMG) .
+	touch $@
+
+.PHONY: run-dev-container
+run-dev-container: .dev-container
```

Comment:
What is the motivation to create a container with all the dependencies but without the clusterlink images

Reply:
It is a container that can be used to develop in isolation (i.e., without installing anything into your machine).
The [`CodeSpaces`](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers) button on GitHub works with it, [`vscode`](https://code.visualstudio.com/docs/devcontainers/containers) can create the container for you, etc.

The only additional step is mounting the ClusterLink source code into the container so it can be edited, built, etc. This is done via the Makefile (vscode does it automatically, see above)